tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/RenderingContent/RenderingContentWithUploadFile.spec.ts

```diff
@@ -0,0 +1,40 @@
+import {AliasHelper, test} from '@umbraco/playwright-testhelpers';
+
+const contentName = 'Test Rendering Content';
+const documentTypeName = 'TestDocumentTypeForContent';
+const templateName = 'TestTemplateForContent';
+const propertyName = 'Test Upload File';
+
+test.afterEach(async ({umbracoApi}) => {
+  await umbracoApi.document.ensureNameNotExists(contentName);
+  await umbracoApi.documentType.ensureNameNotExists(documentTypeName);
+  await umbracoApi.template.ensureNameNotExists(templateName);
+});
+
+const uploadedFileList = [
+  {type: 'an uploaded text file', dataTypeName: 'Upload File', uploadedFileName: 'File.txt', mineType: 'text/plain'},
+  {type: 'an uploaded image', dataTypeName: 'Upload File', uploadedFileName: 'Umbraco.png', mineType: 'image/png'},
```

Comment:
Should this not be Image file?

Reply:
No, as we don't have the specific data type for text file and image file, and the "upload file" data type accept all kinds of files.